docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.4'
+services:
+ sourcecred:
+  container_name: sourcecred
```

Comment:
Using `container_name` and `hostname` limits the ability of docker-compose to handle namespacing and DNS routing for us. I think omitting this is better, as it would allow some easy juggling of different instances (possibly including namespaced named volumes).